test/helpers/web3/preloadTestProvider.js

```diff
@@ -0,0 +1,19 @@
+const initializeTestProvider = require("./initializeTestProvider");
+
+const preloadTestProvider = (options = {}) => {
+  let context = {};
+  before("Setting up web3", async function() {
+    this.timeout(10000);
+    const { accounts, provider, web3 } = await initializeTestProvider(options);
+
+    Object.assign(context, {
```

Comment:
Why object.assign here?

Reply:
File was removed in favor of keep `before` context to the test files.